src/main/kotlin/me/fornever/todosaurus/ui/wizard/TodosaurusWizard.kt

```diff
@@ -215,7 +232,22 @@ class TodosaurusWizard(title: String, private val project: Project, private val
 
         val bottomPanel = JPanel()
 
-        if (mySteps.any { it is MemorableStep }) {
+        if (UserChoiceStore.getInstance(project).getChoiceOrNull() != null) {
+            val forgettableStep = mySteps.firstOrNull { it is ForgettableStep }
+
+            if (forgettableStep is ForgettableStep) {
```

Comment:
_(Minor)_ Might optimize as `mySteps.filterIsInstance<ForgettableStep>().firstOrNull()`, no need for the second cast then.